espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/GetSessions.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.appium.espressoserver.lib.handlers
+
+import java.util.Collections
+
+import io.appium.espressoserver.lib.handlers.exceptions.AppiumException
+import io.appium.espressoserver.lib.model.AppiumParams
+
+import io.appium.espressoserver.lib.model.Session.getGlobalSession
+import java.util.Collections.singletonList
+import java.util.Collections.unmodifiableList
+
+class GetSessions : RequestHandler<AppiumParams, Collection<String>> {
+
+    @Throws(AppiumException::class)
+    override fun handle(params: AppiumParams): Collection<String>? {
+        return if (getGlobalSession() == null) unmodifiableList(emptyList()) else unmodifiableList(listOf(getGlobalSession().id))
```

Comment:
Use `?.let {} ?: run {}` syntax

Reply:
Yeah, I could also do it like:

```kotlin
getGlobalSession()?.let {
  return unmodifiableList(listOf(getGlobalSession().id))
}
return unmodifiableList(emptyList())
```

Reply:
I'm starting to see that a general good practice for Kotlin is to not use the `null` keyword directly.